### PR TITLE
Update SCONE ACE Library file

### PR DESCRIPTION
**Pull request by Mikolaj-A-Kowalski**
This PR makes couple of modifications to the ACE library file. 
- Delimiters are changed from space to semi-colon to allow paths with spaces 
- Limitation on the length of absolute path to an ACE file is removed (in practice determined only by the size of the buffer character to read lines [900 characters at the moment?]) 
- Adds a bash script to search ASCII files for first lines of ACE cards, which writes the library file in SCONE's format. 

If this PR is integrated it will of course break backwards compatibility with existing 'Library Files'. However thanks to the script generating new ones should not be the problem. 

One limitation is that path to the library file itself still cannot contain a space (as a space would confuse the input dictionary parsing... but technically it should be circumvented by using the environmental variable to specify library file path)

### CODE REVIEW

File: NuclearData/ceNeutronData/aceLibrary_mod.f90 [hunk 1]
```diff
@@ -1,8 +1,15 @@
 !!
 !! This module is a factory that produces aceCards for a given nuclide
 !!
+!! Two types of ACE cards can be specified
+!!   Continious energy neutron data e.g. ZZAAA.TTc
+!!   SAB bard e.g. XXXXX.TT.t
```
Comment (valeriaRaffuzzi) [inline NuclearData/ceNeutronData/aceLibrary_mod.f90:6]: card*

File: docs/Installation.rst [hunk 2]
```diff
@@ -261,14 +261,27 @@ its own library file. An example of it is given in *IntegrationTestFiles/testLib
...
-  92233.03c  1       <absolute_path>/9233JEF33.ace
-  1001.03c   4069    <absolute_path>/1001JEF33.ace
+  92233.03c;  1;       <absolute_path>/9233JEF33.ace;
+  1001.03c;   4069;    <absolute_path>/1001JEF33.ace;
   ...
 
 `Line Number` is the line in the file at which a particular data card begins. Each line cannot
-contain more then a single entry. Each component must be space separated. Path can have only 100
-character and contain no spaces.
+contain more then a single entry. Each component must be delimited by a semi-colon.
```
Comment (valeriaRaffuzzi) [inline docs/Installation.rst:269]: than*

**Review by valeriaRaffuzzi — APPROVED** [review]
Looks great, things are neater and the script to produce the libraries was well needed! 
I just commented on a couple of typos

File: NuclearData/ceNeutronData/aceLibrary_mod.f90 [hunk 3]
```diff
@@ -114,19 +122,19 @@ subroutine load(path)
 
       ! Read if line is not empty
       if( len_trim(buffor) /= 0) then
-        associate ( bounds => splitChar(buffor, ' ') )
-          if (size(bounds,2) < 3) call fatalError(Here, 'Ill formatted line: '//trim(buffor))
+        associate ( bounds => splitChar(buffor, DELIM) )
```
Comment (ChasingNeutrons) [inline NuclearData/ceNeutronData/aceLibrary_mod.f90:125]: Another typo: buffer*

**Pull request merged**